test/loader/platforms/platforms.cpp

```diff
@@ -0,0 +1,78 @@
+/*
+ *
+ * Copyright (C) 2023 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ */
+#include <cassert>
+#include <cstdlib>
```

Comment:
Include either cstdlib or stdlib.h, certainly there is no need to include both headers.